libfive/src/solve/bounds.cpp

```diff
@@ -42,8 +42,9 @@ Region<3> findBounds(IntervalEvaluator* eval)
 
```

Comment:
This one was causing the compiler to crash. Breaking it up like this fixed it, but of course there are other ways to do this.

Reply:
I was about to look at this one, how did you find where it crashed the compiler?

Reply:
let's say it was a binary-search commenting out of code